docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -0,0 +1,952 @@
+
+​		**o**   **TIDL has been introduced to support RPC style IPC between applications.**
+
+​		**o**   **Applications can expose RPC style service interface to other applications with TIDL.**
+
+​		**o**   **Native code is generated from the TIDL with tidlc compiler.**
+
+​		**o**   **C, C++, C# languages are supported.**
+
+​		**o**   **tidlc compiler will be released with the Tizen Studio.**
+
+​	**·**        **Watchface Complication Framework, Stylize Editable Feature Support**
+
+​		**o**   **Applications can provides custom complication data with the new watchface complication provider API.**
+
+​		**o**   **Watchface applications can receive data from the complication providers.**
+
+​		**o**   **Watchface applications can register editable design elements and an user will be able to edit the watchface with editor UI.**
+
+​	**·**        **Widget Framework Refactoring**
+
+​		**o**   **Screen connector library has been rewritten in object-oriented design.**
+
+​	**·**        **Package Manager**
+
+​		**o**   **API for changing applications’ icon has been added.**
+
+​		**o**   **Some enum values for fixing errata has been changed.**
+
+​		**o**   **New enum value for extended storage has been added**
+
+**Known Issues**
+
+​	**·**        **N/A**
+
+
+
+## **Window System**
+
+ **New and Changed Features**
+
+​	**·**        wayland
+
+​		**o**   The opensource wayland has been upgraded to       the 1.14.0 version.
+
+- - **o**   The opensource  wayland-protocols has been upgraded to the 1.12 version which has xdg_shell_unstable_v6 protocol. And directory path of xdg-shell protocol unstable version 5 has been moved to protocol/unstable/xdg-shell from protocol/tizen.
+
+  - ·        Extended Wayland Protocols
+
+  - ​	o   tizen_keyrouter protocol has been upgraded to the 2 version to support new event_surface.
+
+  - **·**        **Enlightenment Wayland Display Server**
+
+  - ​	o   **keyrouter functionalities have been moved from Enlightenment modules to the Enlightenment core.**
+
+  - ​	**o**   **Support for xdg_shell_unstable_v6 protocol has been added.**
+
+  - ​	**o**   **e_input and e_input_device have been added to replace input functionalities of the ecore drm.**
+
+  - ​	o   **Dbus policy has applied to org.enlightenment.wm.conf.** 
+
+  - ​	**o**   **The debugging tool has been extended with support for:**
+
+  - ​		**§**  **bgcolor_set:  change the background color of canvas to check compositing of the enlightenment.**
+
+  - ​		**§**  **deiconify_approve: enable and disable deiconify_approve function in run-time.**
+
+  - ​		**§**  **buffer_flush: enable and disable buffer flush function in run-time.**
+
+  - **·**        **Tizen ws Shell** 
+
+  - ​	**o**   **Tizen ws shell has changed to use ecore_wl2 instead of ecore_wayland to support EFL 1.20.**
+
+  - **·**        **efl-util**
+    ​	**o**   efl-util has changed to use ecore_wl2 instead       of ecore_wayland to support EFL 1.20.

```

Comment:
we can remove **o** 
efl-util has changed to use ecore_wl2 instead of ecore_wayland to support EFL 1.20.

Reply:
can not catch the point. 
refering to the original doc. 
"efl-util has changed to use ecore_wl2 instead of ecore_wayland to support EFL 1.20." is listed on the subject "efl-util".